ansible/roles/rhel_zero_touch_user/tasks/copy-files.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Copy user's file
+  when: rhel_zero_touch_user_copy_user_files is defined
+  block:
+    - name: Copy user's file to resource dir
+      ansible.builtin.copy:
+        src: "{{ __file_r.file }}"
+        dest: >
+          /home/{{ rhel_zero_touch_user_name }}/
+          {{ rhel_zero_touch_user_resource_dir_name }}/
+          {{ __file_r.resource }}
```

Comment:
Same here... the `>` quote leaves spaces in the file path and a trailing newline.